package.json

```diff
@@ -14,6 +14,8 @@
     "dev:amo:no-proxy": "better-npm-run dev:amo:no-proxy",
     "dev:disco": "better-npm-run dev:disco",
     "eslint": "eslint .",
+    "flow:check": "flow check",
+    "flow:dev": "chokidar .flowconfig flow/ src/ tests/ -i flow/logs/flow.log -c 'flow status' --initial",
```

Comment:
I forget, is this safe to run if you don't have `chokidar` installed globally? I'm rusty on my `npm run` command knowledge. Maybe it's okay?